assets/html/documenter.css

```diff
@@ -428,3 +432,98 @@ article section.docstring a.source-link {
     background-color: transparent;
     padding: 0;
 }
+
+@media only screen and (max-width: 768px) {
+    div#topbar {
+        display: block; /* is mobile */
+        position: fixed;
+        width: 100%;
+        height: 2.7em;
+        background-color: white;
+        box-shadow: 0 1px 3px rgba(0,0,0,.26);
+        z-index: 2;
+        top: 0em;
+        -webkit-transition-property: top; /* Safari */
+        -webkit-transition-duration: 0.3s; /* Safari */
+        transition-property: top;
+        transition-duration: 0.3s;
+    }
+
+    div#topbar span {
+        position: fixed;
+        width: 80%;
+        height: 1.8em;
+        overflow: hidden;
+        margin-top: 0.6em;
+        margin-left: 1em;
+    }
+
+    div#topbar a.fa-bars {
+        float: right;
+        padding: 0.4em 0.5em 0.2em 1em; /* top right bottom left */
+        font-size: x-large;
+    }
+
+    div#topbar a.fa-bars:visited {
+        color: #3091d1;
+    }
+
+    div#topbar.headroom--unpinned.headroom--not-top.headroom--not-bottom {
+        top: -3em;
+        -webkit-transition-property: top; /* Safari */
+        -webkit-transition-duration: 0.7s; /* Safari */
+        transition-property: top;
+        transition-duration: 0.7s;
+    }
+
+    nav.toc {
+        position: fixed;
+        overflow-y: scroll;
+        width: 16em;
+        left: -16em;
+        -webkit-overflow-scrolling: touch;
+        -webkit-transition-property: left; /* Safari */
+        -webkit-transition-duration: 0.3s; /* Safari */
+        transition-property: left;
+        transition-duration: 0.3s;
+        -webkit-transition-timing-function: ease-out; /* Safari */
+        transition-timing-function: ease-out;
+        z-index: 3;
+    }
+
+    nav.toc.show {
+        left: 0em;
+    }
+
+    article {
+        margin-left: 0em;
+        padding-left: 0.9em;
+        padding-right: 0.9em;
+        overflow-wrap: break-word;
+        z-index: 1;
+    }
+
+    article table {
```

Comment:
Also, why do we have this one exactly? Maybe you could add a few comments here to explain why/how we need to give some elements special treatment to make sure they render properly on mobile (i.e. so that someone not familiar with mobile UX could get up to speed quickly).